pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -633,3 +638,32 @@ func validateTagAttachment(validationCtx *validationContext, reference vim25type
 	}
 	return errors.New(strings.Join(errs, ","))
 }
+
+// validateClusterComputeResourceHasHosts checks the ClusterComputeResource to see if there are any attached hosts.  Without
+// hosts, the check for networks will fail.  We want the message to be clear if this is the cause so
+// users know what to address.  This only returns an error if unable to verify or if no ComputeResources
+// are found.
+func validateClusterComputeResourceHasHosts(ctx context.Context, datacenter string, cluster string, finder Finder) error {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	path := fmt.Sprintf("/%s/host/%s", datacenter, cluster)
```

Comment:
Cluster in this case is a path already. We shouldn't be making paths

Reply:
take a look at this unmerged pr: https://github.com/openshift/installer/pull/6973/files